assets/sass/sidemenu.scss

```diff
@@ -78,12 +78,12 @@
     }
 }
 
-.o-news_list {
+.o-news__list {
     list-style-type: none;
     list-style-position: outside;
     padding-left: 0;
 
-    &__date {
+    &-date {
```

Comment:
Isn't date rather an element instead of a modifier? At least in the examples of https://getbem.com/naming/, date wouldn't fit.